Update AssetPair.php

Triggsy22 opened the pull request:
asset2 is not necceccary for the api request.
And its even possible to send more than two assets to the api.

Triggsy22 committed:
Update AssetPair.php

Changed ValueObjects/AssetPair.php
Asset2 is not necceccary and its even possible to get more than two informations about assetpairs from the api.

butschster commented:
Could you please get links to API where I can read about this case. And also it will be awesome to cover changes with unit tests.

Triggsy22 committed:
Update composer.json

Triggsy22 committed:
Update GetTradableAssetPairsTest.php

changed GetTradableAssetPairsTest.php to show, that more than one assetPair is working.

Triggsy22 commented:
The Api Documentation is: https://docs.kraken.com/rest/#operation/getTradableAssetPairs
Ye, its not obvious that this endpoint is possible with one or more than two AssetPairs.
But just check: https://api.kraken.com/0/public/AssetPairs?pair=ETHXBT%2CBTCUSD%2CXLMUSD%2CDOTUSDT
That are four AssetPairs.
Or: https://api.kraken.com/0/public/AssetPairs?pair=ETHXBT
Thats just one AssetPair.

butschster committed:
Merge branch 'master' into AssetPair-Rework

butschster commented:
Thanks, despite the long delay)
